src/plone/restapi/serializer/dxfields.py

```diff
@@ -29,6 +36,20 @@ def get_value(self, default=None):
                        default)
 
 
+@adapter(IRichText, Schema, Interface)
+@implementer(IFieldSerializer)
+class RichTextFieldSerializer(DefaultFieldSerializer):
```

Comment:
I had to add this serializer because IRichText implements IObject. I haven't find another way to manage this issue